types/flux/utils.d.ts

```diff
@@ -0,0 +1,17 @@
+// Type definitions for Flux Utils 3.1.2
```

Comment:
I thought the linter should have caught this, but only `index.d.ts` should have a header.